### PR TITLE
added the attempt number one

### PR DESCRIPTION
**Pull request by MarchD**
[DEMO LINK](https://MarchD.github.io/react_carousel/)

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -1,19 +1,95 @@
...
+      <div className="wrapper" style={{ width: `${itemWidth * frameSize}px` }}>
+        <ul
+          className="Carousel__list"
+          style={{
+            transform: `translateX(${this.state.transform}px)`,
+            transition: `transform ${animationDuration}ms`,
+          }}
+        >
+          {images.map(img => (
+            <li key={img}>
```
Comment (AlexysUa) [inline src/components/Carousel.js:60]: You can use the numeric key from the image name.
(ex. './img/1.png'.match(/\d/g).join(''))

**Review by AlexysUa — APPROVED** [review]
Great work!

**Review by vsm2k17 — CHANGES_REQUESTED** [review]
great work let's make it even better, add a form which will be controlling the props which you pass into the Carousel component.

File: src/App.js [hunk 2]
```diff
@@ -17,17 +17,105 @@ class App extends React.Component {
...
+              onChange={this.handle}
+            />
+          </label>
+          <label>
+            Size of frame
+            <input
+              name="frameSize"
+              type="text"
+              value={frameSize}
+              onChange={this.handle}
```
Comment (aionova) [inline src/App.js:79]:
```suggestion
              onChange={this.handleInputChange}
```

**Review by aionova — APPROVED** [review]
Awesome!

**Pull request closed without merging**